src/XProfile/VirtoCommerce.XProfile/Queries/GetContactQueryHandler.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using VirtoCommerce.CustomerModule.Core.Model;
+using VirtoCommerce.CustomerModule.Core.Services;
+using VirtoCommerce.ExperienceApiModule.Core.Infrastructure;
+
+namespace VirtoCommerce.ExperienceApiModule.XProfile.Queries
+{
+    public class GetContactQueryHandler : IQueryHandler<GetContactByIdQuery, ContactAggregate>
+    {
+        private readonly IMemberService _memberService;
+        public IDictionary<string, Contact> Results { get; set; } = new Dictionary<string, Contact>();
+
+        public GetContactQueryHandler(IMemberService memberService)
+        {
+            _memberService = memberService;
+        }
+
+        public async Task<ContactAggregate> Handle(GetContactByIdQuery request, CancellationToken cancellationToken)
+        {
+            return new ContactAggregate(
```

Comment:
why you do not use the `IContactAggregateRepository ` here?

Reply:
I think we need to remove it there is GetContactByIdQueryHandler

Reply:
yeah, will remove